### ISSUE TITLE
Multi-page - use of apostrophe

### ISSUE THREAD

**Issue by Ao99** [caobo1994/FourierSeries#14]
An apostrophe followed by an 's' normally shows possession, but do not make a noun plural.
![image](https://user-images.githubusercontent.com/53056165/66706701-1d601900-ed04-11e9-9eb6-36895cf151b7.png)
![image](https://user-images.githubusercontent.com/53056165/66706739-c444b500-ed04-11e9-96d9-d24a8a012fc6.png)
Some examples are shown above. Shouldn't it be GFSs? Is it an academical tradition to write like yours?

**Comment by caobo1994** [comment]
I was told by other professors that when I have to make plural forms of non-traditional words (e.g abbreviations, mathematical expressions), apostrophe is needed to separate the plural 's' from the word itself (most readers will misread when the apostrophe is missing).

**Comment by caobo1994** [comment]
@smiths Could you provide some advice on this topic? Thanks!

**Comment by smiths** [comment]
@caobo1994, I agree with @Ao99.  Apostrophe means possession.  It is an abuse of notation to use it for a plural form.  I think you can write CFSs without ambiguity, since you are using a different font.  There isn't a danger of a reader thinking it is CFS applied to the function s.  Your A_is are trickier.  Try formatting them as $A_i$s and see what it looks like.  If it is really hard to parse, then I guess we can allow an exception to the rule for the purpose of understandability.

**Comment by caobo1994** [comment]
I will adjust and compare the result.
